addon/components/offering-form.js

```diff
@@ -98,37 +93,29 @@ export default Component.extend(ValidationErrorDisplay, Validations, {
   loaded: false,
   'data-test-offering-form': true,
 
-  associatedSchools: computed('cohorts.[]', function(){
-    return new RSVPPromise(resolve => {
-      const cohorts = this.get('cohorts');
-      if (isEmpty(cohorts)) {
-        resolve([]);
-      } else {
-        map(cohorts.toArray(), cohort => {
-          return cohort.get('school');
-        }).then(cohortSchools => {
-          const schools = [];
-          schools.pushObjects(cohortSchools);
-          resolve(schools.uniq().toArray());
-        });
-      }
-    });
-  }),
-  availableInstructorGroups: computed('associatedSchools.[]', function(){
-    return new RSVPPromise(resolve => {
-      this.get('associatedSchools').then(associatedSchools => {
-        map(associatedSchools, school => {
-          return school.get('instructorGroups');
-        }).then(allInstructorGropus => {
-          const flat = allInstructorGropus.reduce((flattened, obj) => {
-            return flattened.pushObjects(obj.toArray());
-          }, []);
-
-          resolve(flat);
-        });
+  associatedSchools: computed('cohorts.[]', async function(){
+    const cohorts = this.get('cohorts');
+    if (isEmpty(cohorts)) {
+      return [];
+    } else {
+      const cohortSchools = map(cohorts.toArray(), cohort => {
```

Comment:
Should be `const cohortSchools = await map`, yes?

Reply:
thanks. fixed.